hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/AllocationFileLoaderService.java

```diff
@@ -495,17 +496,17 @@ private void loadQueue(String parentName, Element element,
       Element field = (Element) fieldNode;
       if ("minResources".equals(field.getTagName())) {
         String text = ((Text)field.getFirstChild()).getData().trim();
-        Resource val =
+        ResourceConfiguration val =
```

Comment:
The major usage of min share is sorting schedulables and preemption. I didn't see many cons. In the other side,
the needs for min share as percentage are limited. It shouldn't a common case that a job/queue says to RM that no matter how much resource you have, I need at least 10% of them. Make min share as percentage isn't necessary as well. Hence, we could just leave it alone.